master/security/service-accounts.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Use service accounts in policy rules
+title: Use service accounts in policy
 description: Use Kubernetes service accounts in policy rules to validate cryptographic identities and/or manage RBAC controlled high-priority rules across teams.
```

Comment:
Use Kubernetes service accounts in policy to validate cryptographic identities and/or manage RBAC controlled high-priority rules across teams.